### ISSUE TITLE
Google Chrome cannot be found

### ISSUE THREAD

**Issue by konstantinbueschel** [ShogunPanda/alfred-chrome#2]
Hi,

I'm getting an error while trying to use the shortcut.
 
Error logs in application window,
 
```
Starting debug for 'Alfred Chrome'

[2017-04-16 22:18:15][ERROR: input.scriptfilter] Code 1: Google Chrome cannot be found.
Nothing to do! :(
[2017-04-16 22:18:15][ERROR: input.scriptfilter] JSON error: JSON text did not start with array or object and option to allow fragments not set. in JSON:
Google Chrome cannot be found.
Nothing to do! :(
```

I am syncing Alfred preferences via Google Drive and installed Google Chrome for all users.

**Comment by aheidelberg** [comment]
I'm getting the same error. There is also [this thread](https://www.alfredforum.com/topic/9473-alfred-chrome-open-url-in-google-chrome-also-with-specific-profile/) on the alfred forums where people are also experiencing the error.

**Comment by ShogunPanda** [comment]
This has been fixed in 4.0.0.

**Comment by konstantinbueschel** [comment]
Great news, thank you!!
